plugins/linux/ifplugin/linuxcalls/link_linuxcalls.go

```diff
@@ -31,38 +31,50 @@
 package linuxcalls
 
 import (
+	"net"
 	"time"
 
-	"github.com/ligato/cn-infra/logging/measure"
 	"github.com/vishvananda/netlink"
 )
 
+// GetLinkFromInterface calls netlink API to get Link type from interface name
+func (handler *netLinkHandler) GetLinkFromInterface(ifName string) (netlink.Link, error) {
```

Comment:
I'd recommend to name this `GetLinkByName` to keep consistency with netlink API and also with ` GetInterfaceByName`.